tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/BrokerRequestTests.cs

```diff
@@ -125,16 +126,56 @@ public void BrokerInteractiveRequestTest()
             }
         }
 
+        [TestMethod]
+        public void BrokerSilentRequestTest()
+        {
+            string CanonicalizedAuthority = AuthorityInfo.CanonicalizeAuthorityUri(CoreHelpers.UrlDecode(TestConstants.AuthorityTestTenant));
+
+            using (var harness = CreateTestHarness())
+            {
+                // Arrange
+                var parameters = harness.CreateAuthenticationRequestParameters(
+                    TestConstants.AuthorityTestTenant,
+                    TestConstants.s_scope,
+                    new TokenCache(harness.ServiceBundle, false),
+                    null,
+                    TestConstants.s_extraQueryParams);
+
+                // Act
+                IBroker broker = harness.ServiceBundle.PlatformProxy.CreateBroker(null);
```

Comment:
By doing this you are using the NullBroker, an implementation that does nothing and that is used on platforms that do not support a broker, i.e. .net

But really we want a where we mock the broker, so that it "can be invoked" and then assert that we get an Access Token back. 

See example test written for iOS : https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/0a65d98f0ed42010dd401201b120daf1e31cdf0f/tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/InteractiveRequestTests.cs#L200